spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -140,6 +140,12 @@ sealed class SpdxExpression {
     fun isValid(strictness: Strictness = Strictness.ALLOW_CURRENT): Boolean =
         runCatching { validate(strictness) }.isSuccess
 
+    /**
+     * Return true if this expression offers a license choice. This is only true, if this expression contains the
```

Comment:
Nit: I believe there should be no comma after `true` grammar-wise. Also, maybe instead of "This is only true" better say "This can only be true".

Reply:
Done.